pyodide_build/recipe/builder.py

```diff
@@ -308,6 +308,23 @@ def ignore(path: str, names: list[str]) -> list[str]:
 
         self.src_dist_dir.mkdir(parents=True, exist_ok=True)
 
+    def _reproducible_filter(
```

Comment:
Doesn't this need to be a `@staticmethod` or `@classmethod`? And it has no self param so I assume it's currently crashing?

Reply:
Yes, also it raises an error because I didn't get the arguments for the filter right. Let me correct that and trigger the integration tests again.

Reply:
Never mind, they were correct here – fixed it.

Reply:
This is a method on `RecipeBuilder` right? So it should take a first argument called `self`? Or be marked `@staticmethod`? Or am I misreading it?

Reply:
Github UI should say "outdated" then =(